app/lib/actions/katello/content_view_version/create_import_history.rb

```diff
@@ -0,0 +1,33 @@
+module Actions
+  module Katello
+    module ContentViewVersion
+      class CreateImportHistory < Actions::EntryAction
+        input_format do
+          param :content_view_version_id, Integer
+          param :path, String
+          param :metadata, Hash
+          param :cv_name, String
+        end
+
+        output_format do
+          param :import_history_id, Integer
+        end
+
+        def run
+          user_full_name = "#{User.current.firstname} #{User.current.lastname}"
```

Comment:
https://github.com/theforeman/foreman/blob/develop/app/models/user.rb#L218
```suggestion
          user_full_name = User.current.to_label
```